tikv-client-store/src/lib.rs

```diff
@@ -1,32 +1,35 @@
 // Copyright 2018 TiKV Project Authors. Licensed under Apache-2.0.
+#![feature(type_alias_impl_trait)]
 
 #[macro_use]
 extern crate log;
 
 mod errors;
 pub mod region;
-mod store_builder;
 
 pub use crate::errors::{HasError, HasRegionError};
 #[doc(inline)]
 pub use crate::region::{Region, RegionId, RegionVerId, StoreId};
-#[doc(inline)]
-pub use crate::store_builder::StoreBuilder;
 pub use kvproto::tikvpb::TikvClient;
-pub use tikv_client_common::{
-    security::SecurityManager, stats::tikv_stats, Error, ErrorKind, Key, Result,
-};
+pub use tikv_client_common::{security::SecurityManager, Error, ErrorKind, Key, Result};
 
+use async_trait::async_trait;
 use derive_new::new;
-use futures::{future::BoxFuture, prelude::*};
 use grpcio::{CallOption, Environment};
 use std::{sync::Arc, time::Duration};
+use tikv_client_common::stats::tikv_stats;
 
 /// A trait for connecting to TiKV stores.
 pub trait KvConnect: Sized + Send + Sync + 'static {
     type KvClient: KvClient + Clone + Send + Sync + 'static;
 
     fn connect(&self, address: &str) -> Result<Self::KvClient>;
+
+    fn connect_to_store(&self, region: Region, address: String) -> Result<Store<Self::KvClient>> {
```

Comment:
What is this function used for?

Reply:
Connecting to a store :-) It's a convenience method for users who use this crate directly so that they don't need to deal with the KvClient trait.